app/components/tabs_component/tab_component.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="page-tab govuk-!-margin-right-8 <%= "page-tab__selected" if current_page?(path) %>">
```

Comment:
Clever.

I imagine if we had more conditional logic I would expect two components - `active_tab_component` and `tab_component` - and the check to be done in tabs. But two if statements are just about less complex than that :)

Reply:
The logic for the check would ideally be done in the controller, I guess - but when I tried that I got an error, and as I said, I'm trying to avoid changing too many things in this PR.

Reply:
Yeah, something for later, when we deem we need it :)